src/Sentry.Unity.Editor/ConfigurationWindow/OptionsConfigurationTab.cs

```diff
@@ -17,23 +17,10 @@ public static void Display(ScriptableSentryUnityOptions options)
 
         EditorGUILayout.BeginVertical("box");
 
-        EditorGUILayout.HelpBox(
-            "The scriptable options configuration allows you to programmatically modify Sentry options." +
-            "\n" +
-            "\n" +
-            "You can use the 'Runtime Configuration Script' to modify options just before Sentry SDK gets " +
-            "initialized. This allows you to access options and functionality otherwise unavailable from the " +
-            "Editor UI, e.g. set a custom BeforeSend callback." +
-            "\n" +
-            "\n" +
-            "Use the 'Build Time Configuration Script' in case you need to change build-time behavior, " +
-            "e.g. specify custom Sentry-CLI options or change settings for native SDKs that start before the " +
-            "managed layer does (such as Android, iOS, macOS).",
-            MessageType.Info);
-
-        EditorGUILayout.Space();
-
-        EditorGUILayout.HelpBox("Deprecated and will be removed in a future version.\nPlease use the 'Option Config Script' below instead.", MessageType.Warning);
+        EditorGUILayout.HelpBox("The Runtime/BuildTime scriptable objects have been deprecated and will be removed in a future version." +
```

Comment:
Should we link to someplace (docs?) that explains in more details what's going on?

If I see a deprecation I always worry about what that means to me: "OK so it won't work anymore what I did before?"

Could we show this message only if it's an upgrade, where they have the old, 2 files thingy. But not for new installs where there are no files yet created? (avoid confusion to new users and don't scare old users?)

Reply:
> Could we show this message only if it's an upgrade, where they have the old, 2 files thingy. But not for new installs where there are no files yet created? (avoid confusion to new users and don't scare old users?)

Great point. Made it so.